.travis.yml

```diff
@@ -199,18 +199,22 @@ script:
               echo -e '+\057\057\057\057 MAKE TEST FAILED'; false;
           fi;
       else
-          if $make build_tests; then
+          if $make build_tests >~/build.out 2>&1; then
               echo -e '+\057\057\057\057\057 MAKE BUILD_TESTS OK';
           else
-              echo -e '+\057\057\057\057\057 MAKE BUILD_TESTS FAILED'; false;
+              echo -e '+\057\057\057\057\057 MAKE BUILD_TESTS FAILED';
+              cat ~/build.out
+              false;
           fi;
       fi
     - if [ -n "$DESTDIR" ]; then
           mkdir "$top/$DESTDIR";
-          if $make install install_docs DESTDIR="$top/$DESTDIR"; then
+          if $make install install_docs DESTDIR="$top/$DESTDIR" >~/install_docs.txt 2>&1 ; then
```

Comment:
And similarly, let me suggest `install.log`

Reply:
Changed as you suggested.